Orange/data/_contingency.pyx

```diff
@@ -1,54 +0,0 @@
-#cython: embedsignature=True
```

Comment:
I think https://github.com/biolab/orange3/pull/1397 might be using this. What is the proposed alternative?